llvm/lib/Support/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ endif( MSVC OR MINGW )
 # Delay load shell32.dll if possible to speed up process startup.
 set (delayload_flags)
 if (MSVC)
-  set (delayload_flags delayimp -delayload:shell32.dll -delayload:ole32.dll)
+  set (delayload_flags $<$<NOT:$<LINK_LANGUAGE:Swift>>:"SHELL:delayimp -delayload:shell32.dll -delayload:ole32.dll">)
```

Comment:
What does the SHELL: do here?

Reply:
The `SHELL:` ensures that the `-delayload` would not be de-duplicated if it is split up.  By default CMake will de-duplicate link arguments, and the `SHELL` modifier tells it to pass it through unaltered.

Reply:
You dropped the SHELL line now, is that because it didn't work with older CMake?

Reply:
Yes, it didn’t work, and currently the list doesn’t have a repeated word so we can get away with it.